README.adoc

```diff
@@ -0,0 +1,208 @@
+endif::[]
+
+
+
+== Overview
+
+This is the server component of the AeroGear Mobile Security Service. It is a RESTful API that allows developers to view, enable and disable specific versions of applications on demand, with the information stored in a PostgreSQL database.
+
+== Prerequisites
+
+.Prerequisites
+|===
+|https://golang.org/doc/install[Install Golang]
+|https://github.com/golang/go/wiki/SettingGOPATH[Ensure the $GOPATH environment variable is set]
+|https://docs.docker.com/compose/install/[Install the dep package manager]
+|https://docs.docker.com/compose/install/[Install Docker and Docker Compose]
+|===
+
+See the https://aerogear.org/community/#guides[Contributing Guide] for more information.
+
+== Getting Started
+
+If you'd like to simply run the entire application in `docker-compose`, follow [these instructions](#Run-Entire-Application-with-Docker-Compose).
+
+Golang projects are kept in a [workspace](https://golang.org/doc/code.html#Workspaces) that follows a very specific architecture. Before cloning this repo, be sure you have a `$GOPATH` environment variable set up.
+
+:numbered:
+=== Clone the Repository
+
+[source,shell]
+----
+git clone git@github.com:aerogear/mobile-security-service.git $GOPATH/src/github.com/aerogear/mobile-security-service
+----
+
+=== Install Dependencies
+
+[source,shell]
+----
+make setup
+----
+
+NOTE: This is using the `dep` package manager under the hood. You will see the dependencies installed in the `vendor` folder.
+
+=== Start the Server
+
+[source,shell]
+----
+go run cmd/mobile-security-service/main.go
+----
+
+=== Run Entire Application with Docker Compose
+
+This section shows how to start the entire application with `docker-compose`. This is useful for doing some quick tests (using the SDKs) for example.
+
+First, compile a Linux compatible binary:
+
+[source,shell]
+----
+go build -o mobile-security-service cmd/mobile-security-service/main.go
+----
+
+This binary will be used to build the Docker image. Now start the entire application.
+
+[source,shell]
+----
+docker-compose up
+----
+:numbered!:
+
+== Setup and configurations
+
+=== Environment Variables
+
+The **mobile-security-service** is configured using environment variables.
+
+* By default, the application will look for system environment variables to use.
+* If a system environment variable cannot be found, the application will then check the `.env` file in the application root.
+* If the `.env` file does not exist, or if the variable is not defined in the file, the application will use the default value defined in [config.go](./pkg/config/config.go).
+
+=== Add your own .env file
+
+Make a copy of the example file `.env.example`.
+
+[source,shell]
+----
+cp .env.example .env
+----
+
+Now the application will use the values defined in `.env`.
+
+=== Server Configuration
+
+.Server Configuration
+|===
+| Variable | Default | Description
```

Comment:
Headings should be in **bold**

Reply:
Regards should I don't know. But it could definitely ;-) 
Done 👍